docs/tutorials/tut1.rst

```diff
@@ -145,14 +159,22 @@ To observe expiration in action, we'll add an :code:`exp` claim of five minutes
 
   select extract(epoch from now() + '5 minutes'::interval) :: integer;
 
-Go back to jwt.io and change the payload to
+Or in ``bash``:
 
-.. code-block:: json
 
-  {
-    "role": "todo_user",
-    "exp": 123456789
-  }
+.. code-block:: bash
+
+  exp=$(( EPOCHSECONDS + 5*60 ))  # five minutes
+
+  echo $exp
+
+Go back to :ref:`tut1_step3` and change the payload to
+
+.. code-block:: bash
+
+  payload=$(echo -n "{\"role\":\"todo_user\",\"exp\":\"123456789\"}" | _base64)
```

Comment:
```suggestion
  payload=$(echo -n "{\"role\":\"todo_user\",\"exp\":\"123456789\"}" | _base64)
```

This should have the `exp` value, not "123456789".

Reply:
It's on purpose, there's a note on the tutorial below.

Reply:
https://github.com/PostgREST/postgrest/blob/0aa397fa1d5ded01633fe123efe17b3bdd19f63e/docs/tutorials/tut1.rst#L179

Reply:
Not to say the tutorial can't be improved later to reduce copy-pastes, but right now I'm keeping that as is.

Reply:
Oh, got it. At first glance I thought it was saying to add `$exp=$(( EPOCHSECONDS + 5*60 ))...` to the bash file. But yes, I didn't notice the `psql` command before, so it's OK.